File: Serene3/Serene3.Web/Modules/ServerTypes/com/productForm.ts
import { DecimalEditor, ImageUploadEditor, initFormType, IntegerEditor, PrefixedContext, ServiceLookupEditor, StringEditor } from "@serenity-is/corelib";

export interface productForm {
    Name: StringEditor;
    Description: StringEditor;
    Price: DecimalEditor;
    Ceito: ServiceLookupEditor;
    Photo: ImageUploadEditor;
    Type: StringEditor;
    Siplername: StringEditor;
    Entitydata: StringEditor;
    Reviewurl: StringEditor;
    Productquntity: StringEditor;
    Rating: DecimalEditor;
    ReviewCount: IntegerEditor;
}

export class productForm extends PrefixedContext {
    static readonly formKey = 'com.product';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!productForm.init) {
            productForm.init = true;

            var w0 = StringEditor;
            var w1 = DecimalEditor;
            var w2 = ServiceLookupEditor;
            var w3 = ImageUploadEditor;
            var w4 = IntegerEditor;

            initFormType(productForm, [
                'Name', w0,
                'Description', w0,
                'Price', w1,
                'Ceito', w2,
                'Photo', w3,
                'Type', w0,
                'Siplername', w0,
                'Entitydata', w0,
                'Reviewurl', w0,
                'Productquntity', w0,
                'Rating', w1,
                'ReviewCount', w4
            ]);
        }
    }
}

File: Serene3/Serene3.Web/Modules/com/productimage/product/productForm.cs
using Serenity.ComponentModel;
using System;

namespace Serene3.com.Forms;

[FormScript("com.product")]
[BasedOnRow(typeof(productRow), CheckNames = true)]
public class productForm
{
    public string Name { get; set; }
    public string Description { get; set; }
    public decimal Price { get; set; }
    public int Ceito { get; set; }
    public string Photo { get; set; }
    public string Type { get; set; }
    public string Siplername { get; set; }
    public DateOnly Entitydata { get; set; }
    public string Reviewurl { get; set; }
    public string Productquntity { get; set; }
    public double Rating { get; set; }
    public int ReviewCount { get; set; }
}